deps/userns_sandbox.c

```diff
@@ -263,7 +263,7 @@ static void mount_overlay(const char * src, const char * dest, const char * bnam
   mkpath(work);
 
   // Construct the opts, mount the overlay
-  snprintf(opts, sizeof(opts), "lowerdir=%s,upperdir=%s,workdir=%s", src, upper, work);
+  snprintf(opts, sizeof(opts), "userxattr,lowerdir=%s,upperdir=%s,workdir=%s", src, upper, work);
```

Comment:
I'm not quite sure this should be always set.  What would be the a good way forward?  Passing yet another option from the command line for whether to use it?